packages/patternfly-4/gatsby-theme-patternfly-org/surveys/qualtrics.js

```diff
@@ -0,0 +1,7 @@
+(function(){var g=function(e,h,f,g){
```

Comment:
This file doesn't need to be in the theme. Can we move it into the `packages/patternfly-4` folder?